extensions/amp-analytics/0.1/variables.js

```diff
@@ -175,6 +175,10 @@ export class VariableService {
     this.register_('LINKER_PARAM', (name, id) =>
       this.linkerReader_.get(name, id)
     );
+    this.register_(
```

Comment:
nit: small request, can we move the macro to above `LINKER_PARAM`. That one is a special one and I want it to live at the bottom of the list. Thank you